src/extension.ts

```diff
@@ -128,4 +129,6 @@ export async function activate(context: ExtensionContext): Promise<void> {
 
     context.subscriptions.push(languages.registerDocumentLinkProvider(LEAN_MODE,
         new LibraryNoteLinkProvider()));
+
+    return {'infoView': infoView}; // export infoView for other plugins to use
```

Comment:
```suggestion
    return {onNewTacticState: infoView.onNewTacticState}; // export infoView for other plugins to use
```

I'd rather export only the event.